src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -181,7 +181,8 @@ public Uri Endpoint
 
     /// <summary>
     /// Gets a value indicating whether or not the signal-specific path should
-    /// be appended to <see cref="Endpoint"/> when creating export clients.
+    /// be appended to <see cref="Endpoint"/> when creating http-based export
```

Comment:
would it better to say when `OtlpExportProtocol.HttpProtobuf` is used?

Reply:
Tweaked